config/environments/production.rb

```diff
@@ -122,7 +122,7 @@
   config.middleware.insert_before 0, Rack::Cors, logger: (-> { Rails.logger }) do
     allow do
       origins(
-        %r{^(https?:\/\/)?(([a-z0-9-]+\.)?)+sumofus\.org$}i, /sumofus.vercel.app/, /pronto-three.vercel.app/, /peoplevsbig.tech/
```

Comment:
Removed the API access given to a previous org, since we no longer communicate with them